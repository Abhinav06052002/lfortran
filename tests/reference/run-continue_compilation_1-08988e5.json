{
    "basename": "run-continue_compilation_1-08988e5",
    "cmd": "lfortran --continue-compilation --no-color {infile}",
    "infile": "tests/errors/continue_compilation_1.f90",
    "infile_hash": "a71fcc9e485b0040897b1a0aa0bae3e23560aece699750eb19d3536a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-continue_compilation_1-08988e5.stdout",
    "stdout_hash": "bb0fa614e9c3660a91fa273686b41f20148054042f5630d4aa3ff87e",
    "stderr": "run-continue_compilation_1-08988e5.stderr",
    "stderr_hash": "135703d0a73dfccc52d850e2242bdf6a50423c1f0bd66205dafd0bbb",
    "returncode": 1
}