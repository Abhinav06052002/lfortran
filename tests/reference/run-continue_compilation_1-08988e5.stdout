(TranslationUnit
    (SymbolTable
        1
        {
            continue_compilation_1:
                (Program
                    (SymbolTable
                        2
                        {
                            a:
                                (Variable
                                    2
                                    a
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            a1:
                                (Variable
                                    2
                                    a1
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (String 1 1 () PointerString)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))
                                        ((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            a2:
                                (Variable
                                    2
                                    a2
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Logical 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))
                                        ((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            a3:
                                (Variable
                                    2
                                    a3
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))
                                        ((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))
                                        ((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            a5:
                                (Variable
                                    2
                                    a5
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            b:
                                (Variable
                                    2
                                    b
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            b1:
                                (Variable
                                    2
                                    b1
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))
                                        ((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            b4:
                                (Variable
                                    2
                                    b4
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))
                                        ((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))
                                        ((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))
                                        ((IntegerConstant 1 (Integer 4) Decimal)
                                        (IntegerConstant 3 (Integer 4) Decimal))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            b5:
                                (Variable
                                    2
                                    b5
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 8)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            c5:
                                (Variable
                                    2
                                    c5
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            i:
                                (Variable
                                    2
                                    i
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            i1:
                                (Variable
                                    2
                                    i1
                                    []
                                    Local
                                    (IntegerConstant 2 (Integer 4) Decimal)
                                    (IntegerConstant 2 (Integer 4) Decimal)
                                    Parameter
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            y1:
                                (Variable
                                    2
                                    y1
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Real 8)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            z1:
                                (Variable
                                    2
                                    z1
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Real 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    continue_compilation_1
                    []
                    [(Assignment
                        (Var 2 a5)
                        (IntegerConstant 8 (Integer 4) Decimal)
                        ()
                    )
                    (Assignment
                        (Var 2 b5)
                        (IntegerConstant 12 (Integer 8) Decimal)
                        ()
                    )
                    (Assignment
                        (Var 2 c5)
                        (IntegerConstant 2 (Integer 4) Decimal)
                        ()
                    )
                    (DoLoop
                        ()
                        ((Var 2 i)
                        (IntegerConstant 1 (Integer 4) Decimal)
                        (IntegerConstant 3 (Integer 4) Decimal)
                        ())
                        [(Print
                            (StringFormat
                                ()
                                [(Var 2 i)]
                                FormatFortran
                                (String -1 0 () PointerString)
                                ()
                            )
                        )]
                        []
                    )
                    (Assignment
                        (Var 2 i1)
                        (IntegerConstant 3 (Integer 4) Decimal)
                        ()
                    )
                    (Print
                        (StringFormat
                            ()
                            [(Var 2 i1)]
                            FormatFortran
                            (String -1 0 () PointerString)
                            ()
                        )
                    )]
                )
        })
    []
)
